src/OpenRiaServices.Tools/Test/ClientClassLib/ClientClassLib.csproj

```diff
@@ -1,6 +1,6 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFrameworks>net472;net6.0</TargetFrameworks>
+    <TargetFrameworks>net472;net6.0;net8.0</TargetFrameworks>
```

Comment:
should we do just net6 or just net8 ? for these class libraries

Reply:
Seems to be needed for some tests when loading project in MsBuildHelper

Reply:
Does it work if you revert the change to msbuildhelper? Otherwise let it be

Reply:
OK, I had a really quick look and it seems the tests adds "itself" as a reference assembly during compilation, 
and it uses OpenRiaServices.Client.Web.csproj to find the system references. (.dlls to use for compilation)

If the test tools reference a newer .net version than client.web it will fail.

possible workarounds:
1. Set reference assembiels based on `this.GetType().Assembly.GetReferencedAssemblies()`
 This should probably be something similar to "add or replace" where any references from tools test (or highest version) wins
2. Keep code as is (maybe consider chaning msbuildhelper to use [TargetFrameworkAttribute](https://stackoverflow.com/questions/72176542/how-to-convert-between-tfm-target-framework-moniker-and-frameworkname) or just highest possible .net version (use `regex `\.net(\d+\.\+)` or with a simple fallback similar to old code)
3. Keep the code as it is

Reply:
I pushed a variant of 2, se what you think about it and drop it if it does not work fine

Reply:
A better solution might be to just move net8.0 before .net6.0 i client.web.csproj 

**If it work s**  that way the msbuildhelper changes should not be needed.